src/pages/hardware/developingAnIOX/messagingProtocol.tsx

```diff
@@ -0,0 +1,1361 @@
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>0-1</td>
+                    <td>0x0003</td>
+                </tr>
+                <tr>
+                    <td>2</td>
+                    <td>Unused</td>
+                </tr>
+            </tbody>
+        </table>
+        <h3>Information Type 4 - Request VIN Message</h3>
+        <p>An IOX uses this message to request the vehicle's VIN from the GO device. The GO device responds with a GO Multi-frame Data (0x27) - Type 3 message.</p>
+        <table>
+            <thead>
+                <tr>
+                    <th>Parameter Type</th>
+                    <th>Description</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>0-1</td>
+                    <td>0x0004</td>
+                </tr>
+                <tr>
+                    <td>2</td>
+                    <td>Unused</td>
+                </tr>
+            </tbody>
+        </table>
+        <h3>Information Type 12 - Request Identification Of Go Device, Versions</h3>
+        <p>Supported from protocol version 1.1.</p>
+        <p>Sent from the IOX to the GO device requesting the identification information. The GO device responds with a GO Multi-Frame Data (0x27) - Type 12 message.</p>
+        <table>
+            <thead>
+                <tr>
+                    <th>Parameter Type</th>
+                    <th>Description</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>0-1</td>
+                    <td>0x000C</td>
+                </tr>
+                <tr>
+                    <td>2</td>
+                    <td>
+                        Request info:
+                        <br></br>
+                        0 = GO serial number
+                        <br></br>
+                        1 = GO firmware version
+                        <br></br>
+                        2 = IOX protocol version
+                    </td>
+                </tr>
+            </tbody>
+        </table>
+        <h3>GO Status Information (0x26)</h3>
+        <p>Sent from the GO device to the IOX to pass information the IOX may need. This is a broadcast message. It is sent once any corresponding information type changes.</p>
+        <h4>Payload</h4>
+        <table>
+            <thead>
+                <tr>
+                    <th>Byte #</th>
+                    <th>Byte Description</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>0-1</td>
+                    <td>Information Type</td>
+                </tr>
+                <tr>
+                    <td>2-7</td>
+                    <td>Optional bytes that depend on the type</td>
+                </tr>
+            </tbody>
+        </table>
+        <h4>Information Type 0 - Ignition</h4>
+        <table>
```

Comment:
we should have tables embedded in a div with the .table-container class until we come up with the final styling so it can be applied later on